content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -192,6 +192,14 @@ necessary soon anyway. Rolling updates of Deployments also provide a natural
 repacking of EndpointSlices with all Pods and their corresponding endpoints
 getting replaced.
 
+### Deduplication
```

Comment:
`# Duplicate endpoints`

Reply:
Echoing https://github.com/kubernetes/website/pull/23084#discussion_r473289550
```suggestion
# Duplicate endpoints
```

Reply:
Updated the title, did not increase the level, assuming the removal of `##` was not intentional here.